packages/core/src/promptState/promptState.ts

```diff
@@ -12,6 +12,11 @@ export interface PromptState<PT extends PromptTask = PromptTask> {
   taskMetadata: PromptTaskMetadata;
 
   lastReviewTimestampMillis: number;
+  /**
+   * @TJS-type object
+   * @additionalProperties true
+   * @nullable
```

Comment:
Curious: why does nullability need to be declared when the type is `| null`? Just a quirk of `typescript-json-schema`?

Reply:
Yep exactly, a small quirk of `typescript-json-schema`. When we specify the `@TJS-type` it'll completely override any type definition, leaving the `| null` to be forgotten in the process.